integration/app/test-demo/src/pages/Home.tsx

```diff
@@ -0,0 +1,20 @@
+import { Heading } from 'amazon-chime-sdk-component-library-react';
+import React from 'react';
+import { routes } from '../app';
+import TestApp from '../components/TestApp';
+import TestApps from '../components/TestApps';
+
+export const Home: React.FC = () => (
+  <>
+    <Heading level={4}>
+      React Component Library Test Demo
+    </Heading>
+    <TestApps>
+      <TestApp
```

Comment:
and can we call this like TestAppData or something

Reply:
It is actually the `LinkToTestAppAndTestCoverage`. `TestApp` looks good to me. concise

Reply:
Sure, Test App is fine